recipes/lal/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "lal" %}
+{% set version = "6.18.0" %}
+{% set sha256 = "464601c529f5607c251a54843e749bc9bd962055cc04ba21fdf6150e392a0ba2" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.xz
+  url: http://software.ligo.org/lscsoft/source/lalsuite/{{ name }}-{{ version }}.tar.xz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - pkg-config
+    - make
+    - {{ compiler('c') }}
+    - python {{ python }}
+    - bc
+    - swig>=3.0.7
+  host:
+    - gsl
+    - zlib
+    - fftw
+    - hdf5
+    - python
+    - numpy
+    - six
+    - lscsoft-glue
```

Comment:
In order to silence some of the conda build patch warnings we may want to add blas in here.

Reply:
`blas` is a required dependency but we currently get this [through](https://git.ligo.org/lscsoft/lalsuite/blob/c302e263bb5ad38df8209f3096f9db9fa3f89a0c/lal/configure.ac#L212) `gsl`. We are currently investigating how to support multiple different `blas` libraries.

Reply:
@mariusvniekerk, should we add an explicit dependency on `openblas` or `blas` in `lal/meta.yaml`, or is @skymoo's explanation enough?